src/pages/about.jsx

```diff
@@ -1,7 +1,22 @@
-import React from "react";
+import React from 'react';
+import aboutPageContent from '../assets/about-content.json';
```

Comment:
Try to use significant keywords to refer to data it would be great if we name this as 
```
import aboutPageData from ...
```
it helps in quickly helps us in knowing if variables are components,data,functions or actions